script.js

```diff
@@ -0,0 +1,199 @@
+// All the DOM selectors stored as short variables
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
+const sendBtn = document.getElementById('send')
+
+let questionNumber = 1
+
+const botReply = (msg) => {
+  showMessage(msg, 'bot')
+}
+
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+}
+
+const showMessage = (message, sender) => {
+  if (sender === 'user') {
+    chat.innerHTML += `
+      <section class="user-msg">
+        <div class="bubble user-bubble">
+          <p>${message}</p>
+        </div>
+        <img src="assets/user.png" alt="User" />  
+      </section>
+    `
+  } else if (sender === 'bot') {
+    chat.innerHTML += `
+      <section class="bot-msg">
+        <img src="assets/bot.png" alt="Bot" />
+        <div class="bubble bot-bubble">
+          <p>${message}</p>
+        </div>
+      </section>
+    `
+  }
+  chat.scrollTop = chat.scrollHeight
+}
+
+// Question Sequence
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
+
+  if (questionNumber === 1) {
+    userReply(message)
+    input.value = ''
+    setTimeout(() => ovenQuestion(message), 1000)
+  } else if (questionNumber === 2) {
+    userReply(message)
+    setTimeout(() => chocolateLevel(message), 1000)
+  } else if (questionNumber === 3) {
+    userReply(message)
+    setTimeout(() => howMuchTime(message), 1000)
+  } else if (questionNumber === 4) {
+    userReply(message)
+    setTimeout(() => suggestRecipe(message), 1000)
+  } else {
+    userReply(message)
+    setTimeout(thankYou, 1000)
+  }
+}
+
+// Chat Start
+const greeting = () => {
+  questionNumber = 1
+  botReply(`Welcome! I look forward to finding the perfect dessert for you &#128523 What's your name?`)
+}
+
+const ovenQuestion = (msg) => {
+  questionNumber++
+  botReply(`Nice to meet you, ${msg}. Do you feel like turning up the heat and preparing something in the oven?`
+  )
+
+  inputWrapper.innerHTML = `
+    <button id="yesBtn">Yes, that's hot!</button>
+    <button id="noBtn">No, let's keep it cool.</button>
+    <button id="maybeBtn">I don't know yet...</button>
+  `
+
+  document
+    .getElementById('yesBtn')
+    .addEventListener('click', () => nextQuestion(`Yes, that's hot!`))
+  document
+    .getElementById('noBtn')
+    .addEventListener('click', () => nextQuestion(`Let's keep it cool`))
+  document
+    .getElementById('maybeBtn')
+    .addEventListener('click', () => nextQuestion(`Don't know yet...`))
+}
+
+const chocolateLevel = (oven) => {
+  questionNumber++
+
+  botReply(`${oven}? Alright! How much chocolate do you want in your dessert?`
+  )
+
+  if (oven === `Yes, that's hot!`) {
+    inputWrapper.innerHTML = `
+    <button id="aLotBtn">Obviously A LOT!</button>
+    <button id="alittleBtn">Just the regular dose</button>
```

Comment:
We need to pay great attention to lowercase vs uppercase characters. You get an error message in the console about this, this should be `"aLittleBtn"`